docs/content/en/docs/community/docs_contributing.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Contributing to EKS Anywhere documentation"
+weight: 20
+description: >
+  Guidelines for contributing to EKS Anywhere documentation
+---
+EKS Anywhere documentation uses the [Hugo](https://gohugo.io/categories/fundamentals) site generator and the [Docsy](https://www.docsy.dev/docs/) theme. To get started contributing:
+
+* View the published EKS Anywhere [user documentation](https://anywhere.eks.amazonaws.com/docs/).
+* Fork and clone the [eks-anywhere](https://github.com/aws/eks-anywhere) project.
+* See [EKS Anywhere Documentation](https://github.com/aws/eks-anywhere/tree/main/docs) to set up your own docs test site.
+* See the [General Guidelines](https://github.com/aws/eks-anywhere/blob/main/docs/content/en/docs/community/contributing.md) for contributing to the EKS Anywhere project
+* Create EKS Anywhere documentation [Issues](https://github.com/aws/eks-anywhere/issues) and [Pull Requests](https://github.com/aws/eks-anywhere/pulls).
+
+## Style issues
+
+* **EKS Anywhere**: Always refer to EKS Anywhere as EKS Anywhere and *NOT* EKS-A or EKS-Anywhere.
+* **Line breaks**: Put each sentence on its own line and don’t do a line break in the middle of a sentence. 
+  We are using a modified [Semantic Line Breaking](https://sembr.org/) in that we are requiring a break at the end of every sentence, but not at commas or other semantic boundaries.
+* **Headings**: Use sentence case in headings. So do “Cluster spec reference” and not “Cluster Spec Reference”
+* **Cross references**: To cross reference to another doc in the EKS Anywhere docs set, use relref in the link so that Hugo will test it and fail the build for links not found. Also use relative paths to point to other content in the docs set. For example:
```

Comment:
```suggestion
* **Cross references**: To cross reference to another doc in the EKS Anywhere docs set, use relref in the link so that Hugo will test it and fail the build for links not found. Also, use relative paths to point to other content in the docs set. For example:
```